react/src/component/login/login.tsx

```diff
@@ -236,11 +236,13 @@ export const LoginExtraInfoForm = (props: { onComplete: ({displayName}) => void
                 <Form.Label column sm="3">
                     Your full name
                 </Form.Label>
-                <Col sm="9">
+                <Col sm="7">
```

Comment:
The "7" vs "9" should be encapsulated. This is going to be another rabbit hole of layout management. The math should be done at some higher level that counts columns and Form fields.